cmd/providers.go

```
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/bionic-dev/bionic/providers"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/spf13/cobra"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+)
+
+var providersCmd = &cobra.Command{
+	Use:   "providers",
+	Short: fmt.Sprintf("List providers available for importing using \"bionic %s\"", importCmd.Use),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		githubProviders := map[string]string{}
+
+		resp, err := http.Get("https://raw.githubusercontent.com/bionic-dev/how-to-export-personal-data/main/readme.md")
+		if err == nil {
+			scanner := bufio.NewScanner(resp.Body)
+			for scanner.Scan() {
+				line := scanner.Text()
+				name := strings.TrimPrefix(line, "### ")
+				if name == line {
+					continue
+				}
+
+				var description string
+				for description == "" && scanner.Scan() {
+					description = scanner.Text()
+				}
+				if description != "" {
+					githubProviders[strings.ToLower(name)] = description
+				}
+			}
+
+			if err := resp.Body.Close(); err != nil {
+				return err
```

you should probably ignore this error